src/Symfony/Bridge/Doctrine/Form/Type/EntityType.php

```diff
@@ -18,70 +18,17 @@
 class EntityType extends DoctrineType
 {
     /**
-     * @var ORMQueryBuilderLoader[]
-     */
-    private $loaderCache = array();
-
-    /**
      * Return the default loader object.
      *
-     * @param ObjectManager $manager
-     * @param mixed         $queryBuilder
-     * @param string        $class
+     * @param ObjectManager         $manager
+     * @param QueryBuilder|\Closure $queryBuilder
```

Comment:
the \Closure should not be documented anymore since its deprecated and not passed anymore